assets/constants.ts

```diff
@@ -117,19 +117,19 @@ export = {
     "description": "বাংলা (Bangla)"
   }, {
     "code": "ca",
-    "description": "català (Catalan)"
+    "description": "Català (Catalan)"
   }, {
     "code": "zh",
     "description": "中文 (Chinese)"
   }, {
     "code": "hr",
-    "description": "hrvatski (Croatian)"
+    "description": "Hrvatski (Croatian)"
   }, {
     "code": "cs",
-    "description": "čeština (Czech)"
+    "description": "Čeština (Czech)"
```

Comment:
here it should be `"čeština (Czech)"` (https://cs.wiktionary.org/wiki/%C4%8De%C5%A1tina)